lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -314,22 +315,39 @@ sub _stage_cromwell_outputs {
 
     my $outputs = $output_result->{outputs};
 
+    my $prefix = $self->_determine_output_prefix;
     for my $output_name (keys %$outputs) {
         my $info = $outputs->{$output_name};
         
-        $self->_stage_cromwell_output($results_dir, $info); 
+        $self->_stage_cromwell_output($results_dir, $info, $prefix);
     }
 
     return 1;
 }
 
+sub _determine_output_prefix {
+    my $self = shift;
+
+    my $build = $self->build;
+
+    my $prefix;
+    my (@prefix_input) = grep { $_->name eq 'output_prefix' } $build->inputs;
```

Comment:
Can there be multiple prefixes? Or is this just defensive coding?

Reply:
Yes and yes.

Build input need not be unique, so there's nothing in the system that precludes multiple `output_prefix` inputs.  I don't expect there to be more than one in practice, but I wanted to handle the possibility anyway.